metaflow/__init__.py

```diff
@@ -212,6 +212,9 @@ def _can_handle_orig_module():
 # this auto-generates decorator functions from Decorator objects
 # in the top-level metaflow namespace
 _import_plugin_decorators(globals())
+# Setting card import for only python 3.4
```

Comment:
This feels a little out of place; could it not go into `_import_plugin_decorators` or something like that? The mechanism for minimum version is an interesting one and we don't have to solve it now but it doesn't feel like it should belong here. If it is too complicated, we can leave it here but should open an issue to fix it.

Reply:
One reason I did this is that cards are not python 2 compatible and GH started shouting test failures for py-2 because of card imports. Should I remove the module import here?

Reply:
I guess we can leave it but if we no longer support Py2, we can just avoid this but can clean this up later.